frontend/src/containers/MetricsStepper/index.tsx

```diff
@@ -222,7 +246,13 @@ const MetricsStepper = () => {
       selectedSourceControls.every(({ branches }) => !isEmpty(branches)) &&
       getDuplicatedSourceControlIds(sourceControlConfigurationSettings).length === 0
     );
-  }, [getDuplicatedSourceControlIds, metricsConfig.sourceControlConfigurationSettings]);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
no `// eslint-disable-next-line react-hooks/exhaustive-deps`

Reply:
done

Reply:
when I remove the `// eslint-disable-next-line react-hooks/exhaustive-deps`, I must change dependents of the `dateRange` to `storeContext`. But when I use `storeContext`, it seems that there are occasional problems, so I will use the `// eslint-disable-next-line react-hooks/exhaustive-deps` now